.github/workflows/build-test-publish.yml

```
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [ '3.9', '3.10', '3.11', '3.12' ]
+        python-version: [ '3.10', '3.11', '3.12', '3.13' ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)